README.md

```diff
@@ -44,7 +44,7 @@ Output will be in `build.release\bin\Release\`
 $ git submodule update --init
 $ brew install gcc6
 $ ./submodules/vcpkg/bootstrap-vcpkg.sh
-$ ./submodules/vcpkg/vcpkg install cpprestsdk[websockets]
+$ ./submodules/vcpkg/vcpkg install "cpprestsdk[websockets]"
```

Comment:
should this also be done in the linux section?

Reply:
Yes I confirm.

Reply:
Can you update it please?